django/contrib/auth/management/__init__.py

```diff
@@ -66,7 +66,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
     except LookupError:
         return
 
-    if not router.allow_migrate(using, Permission):
+    if not router.allow_migrate(using, 'auth', Permission):
```

Comment:
Passing an 'app_label' string and a Model class feels redundant.

In what cases is 'app_label' different from Model._meta.app_label?